migrations/1689237811_convertTextSectionToStructuredTextOnEventsPage.ts

```diff
@@ -0,0 +1,44 @@
+import { Client, buildBlockRecord } from '@datocms/cli/lib/cma-client-node';
+import { parse } from 'parse5'
+import { parse5ToStructuredText } from 'datocms-html-to-structured-text'
+
+export default async function(client: Client): Promise<void> {
+  for await (const page of client.items.listPagedIterator({
+    filter: { type: 'event_item' },
+    version: 'current',
+    nested: true
+  })) {
+    const textSection = await client.itemTypes.find('text_section')
+    const structuredTextSection = await client.itemTypes.find('section_structured_text')
+    const updatedPageBlocks = await Promise.all(
+      // @ts-ignore
+      page.items.map(async block => {
```

Comment:
I didn't figure out how to get it working correctly on Typescript. Unlike the other migrations `page.items` don't support localisation, so it's just a list and not an object.